src/common/ui/smooth-scroll.js

```diff
@@ -0,0 +1,17 @@
+export function createSmoothScroll(el) {
```

Comment:
What about the [built-in smooth scroll](https://developer.mozilla.org/en-US/docs/Web/API/Element/scroll)?

```js
element.scroll({
  top: 100,
  left: 100,
  behavior: 'smooth'
});
```

Reply:
That's much better 😅